core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1540,4 +1540,11 @@ private SqlLibraryOperators() {
           OperandTypes.DATETIME,
           SqlFunctionCategory.TIMEDATE);
 
+  @LibraryOperator(libraries = {SNOWFLAKE, ORACLE})
```

Comment:
Kindly add the TERADATA library, since BITNOT works in Teradata

Reply:
done